package.json

```diff
@@ -149,6 +149,7 @@
     "@storybook/theming": "^6.2.9",
     "@visual-snapshot/jest": "^3.0.0",
     "@wojtekmaj/enzyme-adapter-react-17": "0.6.1",
+    "assert": "^2.0.0",
```

Comment:
Storybook requires assert. See https://gist.github.com/shilman/8856ea1786dcd247139b47b270912324#gistcomment-3681971

We were previously implicitly relying on this

Reply:
nvm just set it to false